docker-compose.yml

```diff
@@ -5,15 +5,17 @@ services:
     volumes:
       - ./files:/input:ro
       - ./output_files:/output
-    command: parsedmarc -c /parsedmarc.ini /input/*
+    command: parsedmarc -c /parsedmarc.ini /input/* --debug
```

Comment:
What is the reason for running with debug flag?

Reply:
Sorry, I just forgot about my fork. That was added just to provide at least some feedback to user. Otherwise there is no info about is it working, or just freezed.